R/readAxivity.R

```diff
@@ -509,7 +510,7 @@ readAxivity = function(filename, start = 0, end = 0, progressBar = FALSE, desire
     frequency_bias = abs(frequency_observed - prevRaw$frequency) / prevRaw$frequency
     if ((raw$blockID != 0 &
          raw$blockID - prevRaw$blockID != 1) |
-        raw$checksum_pass == FALSE |
+        prevRaw$checksum_pass == FALSE |
```

Comment:
we are imputing the previous block's data, so I think we should probably do it when the previous block's checksum is wrong

Reply:
Thanks for spotting that!